sw/device/silicon_creator/rom_ext/rom_ext_common.ld

```diff
@@ -87,7 +103,8 @@ SECTIONS {
     *(.srodata.*)
     *(.rodata)
     *(.rodata.*)
-  } > eflash
+    *(.got*)
```

Comment:
I think I'd prefer it slightly if we just dropped the psuedo-instructions and then removed `.option pic`. I feel like the assembly relying on the GOT might be dangerous (since it isn't necessarily correctly setup during crt0). What do you think?

Reply:
Done! 
I just replaced the `la` pseudoinstructions because it is the only used instruction that has different substitution depending whether there's `.pic` or not.